models/artist.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose')
```

Comment:
This follows the model conventions from the main `Comrad` repository. Right now, the models are similar, but eventually I think they will be just different enough that it will be okay to have both models represented in separate repos.